samples/java/appgengine-java11/simple-server/README.md

```diff
@@ -0,0 +1,70 @@
+# Using the Java Agent on a Simple One File Server App in Google App Engine Standard Java 11
+
+NOTE: This example was copied from
+[appengine-java11/custom-entrypoint](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/appengine-java11/custom-entrypoint)
+and modified for Snapshot Debugger Java agent use.
+
+
+## Setup
+See [Prerequisites](../README.md#Prerequisites).
+
+## Compile the Source With Full Debug Information Enabled
+
+```
+javac -source 11 -target 11 -g Main.java
+```
+
+## Install the Snapshot Debugger Java Agent
+
+```
+mkdir cdbg
+wget -qO- https://github.com/GoogleCloudPlatform/cloud-debug-java/releases/latest/download/cdbg_java_agent_gce.tar.gz | tar xvz -C cdbg
+```
+
+## Deploy to App Engine Standard
+
+Examine the app.yaml file, which specifies the -agentpath java option to load
+the agent. Deploy the app with the following:
```

Comment:
I'd point out the app.yaml directly.  See https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet